components/interface/viewCuberSolvesInterface.vue

```diff
@@ -230,13 +230,14 @@ export default {
           },
         })
 
-        solve.is_tracked = data.updateSolve.is_tracked;
+        solve.is_tracked = data.updateSolve.is_tracked
 
-        sharedService.generateSnackbar(
-          this.$root,
-          'Solve ' + (solve.is_tracked ? 'Re' : 'Un') + '-tracked',
```

Comment:
I feel like these kinds of stuff are a lot clearer in their own variable I don't mind duplicating the text a bit, so I changed it to: `const message = solve.is_tracked ? 'Solve tracked' : 'Solve un-tracked'`.
But just FYI, you can also use template literals to do these kinds of string concatenation: `` `Solve ${solve.is_tracked ? 'Re' : 'Un'}-tracked` ``. In this specific case it doesn't change too much, but template literals generally look better and are overall more flexible (can have linebreaks, no need to escape quotes and double-quotes, etc), so people generally prefer that over the plain old string concatenation 😃

Reply:
Good call, yeah that all makes sense to me. Haven't really used much of template literals before, but I'll try to use them more in the future.